samples/interactive-tutorials/src/test/java/events/ImportUserEventsBigQueryTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package events;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import util.StreamGobbler;
+
+public class ImportUserEventsBigQueryTest {
+
+  private String output;
+
+  @Before
+  public void setUp() throws IOException, InterruptedException, ExecutionException {
```

Comment:
Please refer to this [example](https://github.com/googleapis/java-bigquery/blob/001302bbfd3d1e071bb48ad2a783183fd48f6059/samples/snippets/src/test/java/com/example/bigquery/CreateDatasetIT.java#L54) for sample integration test.

Reply:
Changed testing approach.